<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Abdul-PC" ignored="0" name="Homepagetests" tests="1" failures="1" timestamp="06 Oct 2020 04:37:45 GMT" time="258.927" errors="0">
  <testcase name="homeScreen" time="52.112" classname="framework.Hybrid.HomePage">
    <failure type="java.lang.AssertionError" message="expected [An Academy to Learn Ern &amp;amp; Shine  in your QA Career] but found [An Academy to Learn Earn &amp;amp; Shine  in your QA Career]">
      <![CDATA[java.lang.AssertionError: expected [An Academy to Learn Ern & Shine  in your QA Career] but found [An Academy to Learn Earn & Shine  in your QA Career]
at framework.Hybrid.HomePage.homeScreen(HomePage.java:37)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- homeScreen -->
</testsuite> <!-- Homepagetests -->
